3_javascript/javascript1/week1/main.js

```diff
@@ -38,4 +38,15 @@ function  calculator(wide, deep, high, gardenSizeInM2, price){
 calculator(8, 10, 10, 100, 2500000)
 
 //function for Julia. Julia pays less than market value. ^.^
-calculator(5, 11, 8, 70, 1000000)
\ No newline at end of file
+calculator(5, 11, 8, 70, 1000000)
+
+
+//Ez Namey
+const firstWords = ["Cutie", "Tabby", "Chonky", "Grumpy", "Liquid", "Fluffy", "Squishable", "One-Brain-Cell", "Hangry", "Sleepy"]
+const secondWords = ["Cat", "Goat", "Giraffe", "Lion", "Tiger", "Cow", "Horse", "Duck", "Whale", "Leopard"]
+const randomNumber = Math.floor(Math.random() * 10);
+
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber]
+console.log(startupName)
+
+console.log("The startup: " + startupName + " contains " + startupName.length + " characters.")
```

Comment:
your startupName.length will give you always number with an space between two words, wich you adding here : `const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber] `.
So to get precise number of characters without space, you can use for example : `console.log("The startup: " + startupName + " contains " + (startupName.length-1) + " characters.")` :)